izumi-reflect/izumi-reflect/src/main/scala-2/izumi/reflect/ReflectionUtil.scala

```diff
@@ -56,6 +56,13 @@ private[reflect] object ReflectionUtil {
       case t: Universe#RefinedTypeApi =>
         t.parents.forall(allPartsStrong(outerTypeParams, _)) &&
         t.decls.toSeq.forall((s: Universe#Symbol) => s.isTerm || allPartsStrong(outerTypeParams, s.asType.typeSignature.dealias))
+      case e: Universe#ExistentialTypeApi =>
+        allPartsStrong(outerTypeParams, e.underlying) &&
+          e.underlying.typeArgs.forall(t => t.typeSymbol.typeSignature match {
+            case tb: Universe#TypeBounds => allPartsStrong(outerTypeParams, tb.hi) && allPartsStrong(outerTypeParams, tb.lo)
```

Comment:
```suggestion
            case tb: Universe#TypeBoundsApi => allPartsStrong(outerTypeParams, tb.hi) && allPartsStrong(outerTypeParams, tb.lo)
```
To avoid warning `abstract type pattern scala.reflect.api.Universe#TypeBounds is unchecked since it is eliminated by erasure`